src/instructlab/rag/document_store_factory.py

```diff
@@ -0,0 +1,43 @@
+# Standard
+import logging
+
+# First Party
+from instructlab.rag.document_store import DocumentStoreIngestor, DocumentStoreRetriever
+
+logger = logging.getLogger(__name__)
+
+
+def create_document_retriever(
+    document_store_uri: str,
```

Comment:
can we add doc strings for all of these for folks unfamiliar with the functionality?